services/analytics-service-base.ts

```diff
@@ -56,6 +62,15 @@ export class AnalyticsServiceBase implements IAnalyticsService {
 		}
 	}
 
+	public async trackAcceptFeatureUsage(settings: { acceptTrackFeatureUsage: boolean }): Promise<void> {
+		try {
+			await this.sendDataToEqatecMonitors(this.acceptUsageReportingAPIKeys,
+				(eqatecMonitor: IEqatecMonitor) => eqatecMonitor.trackFeature(`${this.acceptTrackFeatureSetting}.${settings.acceptTrackFeatureUsage}`));
+		} catch (e) {
+			this.$logger.trace(`Analytics exception: '${e}'`);
```

Comment:
In case an exception is raised here we'd trace `Analytics exception: '[object Object]'`